tests/xilinx_test.sh

```diff
@@ -124,6 +124,10 @@ run_all() {
 
     # Views
     run_multi_sample fpga/reshape_view_fpga 0 "view_fpga" "reshp_np_1" "reshapedst_1"
+
+    # RTL cores
+    DACE_compiler_xilinx_mode="hardware_emulation"
+    LIBRARY_PATH=$LIBRARY_PATH:/usr/lib/x86_64-linux-gnu run_sample rtl/hardware_test floating_point_vector_plus_scalar 0 1
```

Comment:
Maybe we can move this to CMake? Use [`find_library`](https://cmake.org/cmake/help/latest/command/find_library.html) to look for the library that hardware emulation needs and pass it to DaCe through, e.g., `DACE_XILINX_LIBRARY_PATH`, which we can export in `dace_init` as an environment variable before running the kernel.

Reply:
As discussed, this should be kept, for now.

Reply:
Keeping until no one is using anything older than 2020.2 anymore, as discussed.